app/jobs/correct_response_needed_values_job.rb

```diff
@@ -5,7 +5,8 @@ def perform
       incoming_sms_times = client.incoming_text_messages.pluck(:received_at)
       incoming_email_times = client.incoming_emails.pluck(:received_at)
       incoming_doc_times = Document.where(uploaded_by: client).pluck(:created_at)
-      incoming_times = (incoming_sms_times + incoming_email_times + incoming_doc_times).compact
+      incoming_signature_doc_times = Document.where(document_type: DocumentTypes::CompletedForm8879.key).pluck(:created_at)
+      incoming_times = (incoming_sms_times + incoming_email_times + incoming_doc_times + incoming_signature_doc_times).compact
```

Comment:
This script backfills the "needs response" times based on historical interactions, in order to clear out false positives and more correctly represent previous client interactions. Since we will begin counting Signed 8879 forms as an incoming interaction, I'll need to include it in the set of client-initiated events.